virttest/test_setup.py

```diff
@@ -1379,33 +1379,21 @@ def remove_driver(self, driver=None):
         return True
 
     @error_context.context_aware
-    def modprobe_driver(self, driver=None, option=None):
+    def modprobe_driver(self, driver=None):
         """
         Method to modprobe driver
 
         :param driver: driver name
-        :param option: modprobe options to driver
         :return: True on success, False on failure
         """
-        msg = "Loading the driver '%s' without option" % driver
-        error_context.context(msg, logging.info)
         if not driver:
             driver = self.driver
+        msg = "Loading the driver '%s' without option" % driver
+        error_context.context(msg, logging.info)
         cmd = "modprobe %s" % driver
-        if not option:
-            option = self.driver_option
-        if driver and driver != "mlx5_core" and ARCH != 'x86_64':
-            if option:
-                cmd += " %s" % option
         if process.system(cmd, ignore_status=True, shell=True):
             logging.debug("Failed to modprobe driver: %s", driver)
             return False
-        if ARCH == 'ppc64le' and self.driver == 'mlx5_core':
-            pf_devices = self.get_pf_mlx()
-            if (self.get_vfs_count() > 0):
-                for PF in pf_devices:
-                    if not self.set_vf(PF, option):
-                        return False
         return True
```

Comment:
As L1403-1408  be removed,  how about change `return True`  to  
```
else:
    return True
```

Reply:
I'd prefer to keep 'return True' only and 'else' is not necessary here. It will return when modprobe driver successfully in L1394.

Reply:
Same,  it is ok for me too.